### PR TITLE
Clone incompletes caused by cache service errors automatically

### PR DESCRIPTION
**Pull request by Martchus**
* It is possible to configure the regex being used
* It is possible to disable this kind of automatic cloning
* See https://progress.opensuse.org/issues/80202#note-15

---

* ~~Based on https://github.com/os-autoinst/openQA/pull/3618 which could be reviewed individually first~~ rebased on master after #3618 was merged
* See last commit for only the additional change

### CODE REVIEW

File: etc/openqa/openqa.ini [hunk 1]
```diff
@@ -71,6 +71,10 @@
 ## Gracefully restart the prefork workers if they reach a certain memory limit (in kB)
 #max_rss_limit = 180000
 
+## Causes jobs reported as incomplete by the worker to be cloned automatically when the
```
Comment (okurz) [inline etc/openqa/openqa.ini:74]: I suggest you also describe the feature in documentation, e.g. similar to how I have done in https://github.com/os-autoinst/openQA/pull/3603/files#diff-71734bf4d91118f92743c7d0db9218c4f24729410e020110884606b5ad6a3878R751 . Could be just the next section after that

Comment (Martchus) [reply etc/openqa/openqa.ini:74]: added

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
sorry, one of my PRs has caused a conflict in docs/Installing.asciidoc

**Comment by Martchus** [comment]
The factory build of `openQA:openQA-devel-test` is currently blocked by chromium/chromedriver. Since everything else passed I'm merging the PR anyways.

**Pull request merged**